spec/unit/puppet/provider/conf_setting/ruby_spec.rb

```diff
@@ -194,6 +194,51 @@ def validate_file(expected_content,tmpfile = tmpfile)
       EOS
       )
     end
+
+    it "should convert a scalar key to a single element array when type is set" do
+      File.open(tmpfile, 'w') do |fh|
+        fh.write('ennui: yes')
+      end
+      validate_file('ennui: yes')
```

Comment:
absolutely. i normally wouldn't do this but i was a stranger in a strange land and any bit of sanity checking was helping me. i left em in because i figured they wouldn't hurt.